### PR TITLE
Folders and Metings logs

### PR DESCRIPTION
**Pull request by BarbzCodez**
I added the meetings log and folders, let me know if I missed anything for it

### CODE REVIEW

**Review by dlarocque — COMMENTED** [review]
Can we use one of the commonly used filename conventions such as `kebab-case.md`?

Using spaces in filenames means we have to refer to file names in docs like this `./Documentation/Project%20Proposal.md`.

It also gets funky with the command line, where we need quotes
```bash
$ ls Documentation/
'Meetings Log.md'  'Project Proposal.md'

```

File: README.md [hunk 1]
```diff
@@ -7,3 +7,16 @@
 - Sprint 2
 - Sprint 3
 - Sprint 4
+
+## Coding conventions
+
+-`kebab-case` for doc names
+-`camelCase` for vars
+Do we have a preferred linter?
```
Comment (dlarocque) [inline README.md:15]: ESLint + Prettier with recommended TypeScript, React, Prettier plugins.

Do we need to explicitly define all our coding conventions or do we want to leave that up to the linters and formatters?

Comment (BarbzCodez) [reply README.md:15]: I don't think we need to explicitly define all our coding convention, but I think it is worth it to write down which linters we are gonna use, I'm cool with ESLint and Prettier with default configs

Comment (dlarocque) [reply README.md:15]: Since we don't need to define our coding convention in the readme, should we remove this section?

**Review by lysackm — APPROVED** [review]
LGTM

**Review by dlarocque — APPROVED** [review]

**Comment by dlarocque** [comment]
We should merge all the other PRs into this one, and do one big merge with the final documentation for sprint 1. Otherwise, we'll have to rebase and cherry-pick commits which is no fun

**Pull request merged**